imaginedsf-custom-theme/functions.php

```diff
@@ -14,6 +14,13 @@
 require_once 'inc/custom-post-types.php';
 
 
+/*
+*   Adds ACF Options pages.
+*/
+
+require_once 'inc/options.php';
```

Comment:
So it looks like the `options.php` page creation snuck into previous commit 0e09c409cb748bff58a8bd1eab4d1ad92ee7145c and I can't leave comments on it, so I'll just leave it here.  `options.php` is almost right, but if you look in WP you'll see that there's an additional Options page just called "Options".  That's because you're actually calling the function a fourth time on line 36: the `add_action` WP function calls the function named whatever the 2nd argument is during the hook execution stage of WP's loading referenced by the 1st argument, so it'll just call `acf_add_options_page` with no arguments during the `init` hook.  I don't think you need to use hooks for this, so you can just delete that line.